libcommon/led.c

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright (C) 2022, 2023 - Tillitis AB
+ * SPDX-License-Identifier: GPL-2.0-only
+ */
+
+#include <led.h>
+#include <tk1_mem.h>
+#include <types.h>
+
+static volatile uint32_t *led = (volatile uint32_t *)TK1_MMIO_TK1_LED;
```

Comment:
Wouldn't it make declare `led` as `static volatile uint32_t* const led = (volatile uint32_t *)TK1_MMIO_TK1_LED;` since we don't want to change what `led` is pointing to?